lib/samples/control_annotation_sublayer_visibility/README.md

```diff
@@ -0,0 +1,59 @@
+# Control annotation sublayer visibility
+
+Use annotation sublayers to gain finer control of annotation layer subtypes.
+
+![Image of control annotation sublayer visibility](control_annotation_sublayer_visibility.png)
+
+## Use case
+
+Annotation, which differs from labels by having a fixed place and size, is typically only relevant
+at particular scales. Annotation sublayers allow for finer control of annotation by allowing
+properties (like visibility in the map and legend) to be set and others to be read (like name) on
+subtypes of an annotation layer.
+
+An annotation dataset which marks valves as "Opened" or "Closed", might be set to display the "
+Closed" valves over a broader range of scales than the "Opened" valves, if the "Closed" data is
+considered more relevant by the map's author. Regardless, the user can be given a manual option to
+set visibility of annotation sublayers on and off, if required.
+
+## How to use the sample
+
+Start the sample and take note of the visibility of the annotation. Zoom in and out to see the
+annotation turn on and off based on scale ranges set on the data.
+
+Use the checkboxes to manually set "Open" and "Closed" annotation sublayers visibility to on or off.
+
+## How it works
+
+1. Create a new `ArcGISMapView` with an `ArcGISMapViewController`.
+2. Load a `MobileMapPackage` that contains `AnnotationSublayer`.
+3. Get the sublayers from the map package's layers by calling `sublayer.subLayerContents`.
+4. You can toggle the visibility of each sublayer manually using `sublayer.isVisible` property.
+5. To determine if a sublayer is visible at the current scale of the `ArcGISMapView`,
+   use `sublayer.isVisibleAtScale(ArcGISMapViewController.scale)`, by passing in the map's current
+   scale.
```

Comment:
```suggestion
3. Get the sublayers from the map package's layers by calling `Sublayer.subLayerContents`.
4. You can toggle the visibility of each sublayer manually using `Sublayer.isVisible` property.
5. To determine if a sublayer is visible at the current scale of the `ArcGISMapView`,
   use `Sublayer.isVisibleAtScale(ArcGISMapViewController.scale)`, by passing in the map's current
   scale.
```
This is a very minor thing and not incorrect as such, but for consistency we (Flutter) tend to use the Class name here rather than variable. There are some differences in these parts of the readme's between platforms.

Reply:
Thank you for pointing it out !